File: adapters/infinity/java/com/duckdigit/isam/DisconnectRequest.java
/*==================================================================================================
* Copyright (c) DuckDigit Technologies, Inc.  2008,2009
* ALL RIGHTS RESERVED
* $Header: /home/cvsroot/ai/adapters/infinity/java/com/duckdigit/isam/DisconnectRequest.java,v 1.1 2010/10/19 00:44:26 secwind Exp $
==================================================================================================*/
package com.duckdigit.isam;

public class DisconnectRequest extends IsamRequest
{
	public DisconnectRequest (byte buf [])
		throws IsamException
	{
		super (buf, IsamRequest.DIS);
	}
}

File: adapters/tam/src/disconnectrequest.cpp
/******************************************************************
* Copyright (c) DuckDigit Technologies, Inc.  ALL RIGHTS RESERVED
* $Header: /home/cvsroot/ai/adapters/tam/src/disconnectrequest.cpp,v 1.1 2009/09/20 07:15:31 secwind Exp $
*****************************************************************/
#include <dbclient.h>
#include <disconnectrequest.h>

DisconnectRequest_::DisconnectRequest_ (char *buf)
	: DbRequest_ (buf, DIS)
{
}
